packages/contracts/package.json

```diff
@@ -2,17 +2,21 @@
   "name": "@aragon/core-contracts",
   "version": "0.0.1",
   "description": "Aragon V2 DAO Framework",
-  "main": "index.js",
+  "main": "dist/bundle-cjs.js",
+  "module": "dist/bundle-esm.js",
+  "types": "dist/bundle.d.ts",
```

Comment:
browser build resp. UMD build is missing

Reply:
It should be 8.1.0 (see commit 51896b5348eb248e472250720d39eb1f8f81a309).

Reply:
I'm not sure I follow the idea. `main` is the generic or Node only version, while `module` is the version for ES modules, that browsers could already understand natively.

The concept of a browser having a `<script>` tag loading `mysite.com/node_modules/@aragon/core-contracts-ethers/index-umd.js` to access `window.AragonCoreContractsEthers.xxx` is something I would disencourage/deprecate as much as possible